### ISSUE TITLE
"Pin to profile" style mismatch

### ISSUE THREAD

**Issue by yheuhtozr** [forem/forem#14503]
**Describe the bug**

![image](https://user-images.githubusercontent.com/84892012/129504438-d2e28da5-922d-4082-a2f5-9b0793bdabb7.png)

**To Reproduce**

Open the manager UI of any post of yours.

**Expected behavior**

All blue buttons should have the same font.

**Desktop (please complete the following information):**

- OS, version: Win 10 Home 21H1
- Browser, version: Chrome 92

**Comment by benhalpern** [comment]
Yes, this page still has our older styles. Updating to `crayons` buttons and general styles is a needed improvement.

**Comment by manojnaidu619** [comment]
Can I grab this issue?

**Comment by nickytonline** [comment]
For sure @manojnaidu619 ! Assigning it to you.

**Comment by manojnaidu619** [comment]
I've been getting this error while I'm trying to setup things on local...I run into this below error after running `rails s`
```
Compiling...

...

...

ERROR in ./app/javascript/packs/articleModerationTools.js
Module not found: Error: Can't resolve './flagUserModal' in '/Users/manojnaidu619/Desktop/forem/app/javascript/packs'
 @ ./app/javascript/packs/articleModerationTools.js 30:19-44

ERROR in ./app/javascript/actionsPanel/actionsPanel.js
Module not found: Error: Can't resolve '@utilities/http' in '/Users/manojnaidu619/Desktop/forem/app/javascript/actionsPanel'
 @ ./app/javascript/actionsPanel/actionsPanel.js 19:0-42 76:23-30 156:19-26 260:19-26
 @ ./app/javascript/packs/actionsPanel.js


warning ../../package.json: No license field

  Rendered articles/index.html.erb within layouts/application (Duration: 11781.1ms | Allocations: 14171)
Completed 500 Internal Server Error in 12415ms (ActiveRecord: 46.2ms | Allocations: 135906)

ActionView::Template::Error - Webpacker can't find homePage.js in /Users/manojnaidu619/Desktop/forem/public/packs/manifest.json. Possible causes:
1. You want to set webpacker.yml value of compile to true for your environment
   unless you are using the `webpack -w` or the webpack-dev-server.
2. webpack has not yet re-run to reflect updates.
3. You have misconfigured Webpacker's config/webpacker.yml file.
4. Your webpack configuration is not creating a manifest.
Your manifest contains:
{
  "actionsPanel.js": "/packs/js/actionsPanel-907dc7caee6330fa2200.js",
  "actionsPanel.js.map": "/packs/js/actionsPanel-907dc7caee6330fa2200.js.map",
  "analyticsArticle.js": "/packs/js/analyticsArticle-bd07cc7cedf063f7c180.js",
  "analyticsArticle.js.map": "/packs/js/analyticsArticle-bd07cc7cedf063f7c180.js.map",
   ...
```
Can I get some help here?

**Comment by rhymes** [comment]
@manojnaidu619 this usually happens when a build is updated while the server is running and there's a misalignment between the files installed and the files that are loaded in the webserver. A couple of things:

1. make sure your branch is up to date with main, for example with `git checkout main && git pull --rebase main` and then `git checkout yourbranch && git merge main`
2. Run `bin/setup` to make sure all dependencies are installed and up to date
2. try using `bin/startup` to start the app

Let us know if you need further help :)

**Comment by Radhesh-Sarma** [comment]
@nickytonline Should I take this up and try ?

**Comment by manojnaidu619** [comment]
Thanks @rhymes It worked for me this time
@Radhesh-Sarma I'll have a look into the issue soon and revert back.

**Comment by aitchiss** [comment]
Fixed in #14617
